lib/itamae/resource/gem_package.rb

```diff
@@ -43,6 +45,13 @@ def action_install(action_options)
         end
       end
 
+      def action_uninstall(action_options)
+        if current.installed
+          uninstall!
+          updated!
```

Comment:
I think this line is not necessary because if `attributes.installed != current.installed`, Itamae marks the resource as updated automatically.